admin_guide/manage_scc.adoc

```diff
@@ -429,56 +429,57 @@ To provide additional capabilities:
 === Modify Cluster Default Behavior
 
 To modify your cluster so that it does not pre-allocate UIDs, allows containers
-to run as any user, and prevents privileged containers:
+to run as any user, but prevents privileged containers, grant access to the
+*anyuid* SCC for everyone:
 
-[NOTE]
-====
-In order to preserve customized SCCs during upgrades, do not edit settings on
-the default SCCs other than priority, users, groups, labels, and annotations.
-====
-
-. Edit the *restricted* SCC:
-+
 ----
- $ oc edit scc restricted
+ $ oc adm policy add-scc-to-group anyuid system:authenticated
 ----
 
-. Change `*runAsUser.Type*` to *RunAsAny*.
-
-. Ensure `*allowPrivilegedContainer*` is set to false.
-
-. Save the changes.
-
 To modify your cluster so that it does not pre-allocate UIDs and does not allow
-containers to run as root:
+containers to run as root, grant access to the *nonroot* SCC for everyone:
 
-. Edit the *restricted* SCC:
-+
 ----
- $ oc edit scc restricted
+ $ oc adm policy add-scc-to-group nonroot system:authenticated
 ----
 
-. Change `*runAsUser.Type*` to *MustRunAsNonRoot*.
-
-. Save the changes.
+[WARNING]
+====
+Be very careful with any modifications that have a cluster-wide impact! When an
```

Comment:
@php-coder, if this is still accurate, I'd use this text instead:

Be very careful with any modifications that have a cluster-wide impact. When you
grant an SCC to all authenticated users, as in the previous example, it also
affects Kubernetes and {product-title} components, including web console and
the integrated docker registry. Changes made with this SCC can cause these
components to stop functioning.

Take the same considerations into account when you modify an SCC, such as
granting the *restricted* SCC to everyone. Always prefer creating a custom
SCC and target it to only specific users or groups. This way potential issues
are confined to the affected users or groups and do not impact critical cluster
components.

Reply:
Also, please take a look at the note at ~37 and make sure you're still happy with it.

Reply:
> take a look at the note at ~37

I couldn't find this note. Is it a line number?

> Take the same considerations into account when you modify an SCC, such as
granting the restricted SCC to everyone.

This is not what I had in mind initially:

>> The same considerations should be taken into account when an SCC is being modified (for example, the *restricted* SCC is granted to everyone).

I've tried to say that when you're changing `restricted` SCC you have to think twice because this SCC is already granted to all users by default.

@kalexand-rh How I can express this idea then?

Reply:
@php-coder, you're right. The note's at line 33 now:

> Do not modify the default SCCs. Customizing the default SCCs can lead to issues
when upgrading. Instead, xref:creating-new-security-context-constraints[create
new SCCs].

 Ok. How's this?
> Be very careful with any modifications that have a cluster-wide impact. When you
grant an SCC to all authenticated users, as in the previous example, or modify an
SCC that applies to all users, such as the _restricted_ SCC, it also affects
Kubernetes and {product-title} components, including the web console and
integrated docker registry. Changes made with this SCC can cause these
components to stop functioning.

> Always prefer creating a custom SCC and target it to only specific users or
groups. This way potential issues are confined to the affected users or groups
and do not impact critical cluster components.